gooey/gui/model.py

```diff
@@ -207,8 +208,18 @@ def skipping_config(self):
     return self.build_spec['manual_start']
 
   def is_required_section_complete(self):
-    completed_values = filter(None, [arg.value for arg in self.required_args])
-    return len(self.required_args) == len(completed_values)
+    error_found = False
+    for arg in self.required_args:
+      widget = arg.widget_instance.widget_pack.widget
+      if arg.value:
+        widget.SetBackgroundColour(wx.NullColour)
+      else:
+        if not error_found:
+          error_found = True
+          widget.SetFocus()
+        widget.SetBackgroundColour("Red")

```

Comment:
I think straight red is a little too bold for Gooey's muted palette. Additionally, you keep the font color the same, so it all end up looking a little wonky. 

Could you soften it up a bit? `#EF9A9A` looked pretty good to my eye.

Reply:
Done